### DISCUSSION TITLE
[Bug][Unsupported Version][25921+] taskbar doesn't show system icons

### DISCUSSION THREAD

**Discussion by CorruptionVR** [valinet/ExplorerPatcher#2363] [General]
right side of taskbar no work.
![image](https://github.com/valinet/ExplorerPatcher/assets/78228404/261b6afe-7a0f-4173-b54e-ee2dd4ade3e9)
console line

```shell
Running on Windows 11, OS Build 10.0.25982.1000.
Loaded symbols
Setup user32 functions done
CAppResolverCacheBuilder::_AddUserPinnedShortcutToStart() = 42D40
Patched taskbar transparency in newer OS builds
ITrayUIHost = 30D628
Setup explorer functions done
Setup uxtheme functions done
[AC] rcMonitorAssignment = 992D9
[AC] rcWorkAssignment = 99387
[AC] blockBegin = 992EE
[AC] blockEnd = 99398
[AC] Patched!
[CC] rcMonitorAssignment = 9C7BE
[CC] rcWorkAssignment = 9C88A
[CC] blockBegin = 9C7D6
[CC] blockEnd = 9C89E
[CC] Patched!
[TC] rcMonitorAssignment = 369D94
[TC] rcWorkAssignment = 369E3B
[TC] blockBegin = 369DAB
[TC] blockEnd = 369E4F
[TC] Patched!
[TV] firstCallPrep = 661BAE
[TV] firstCallCall = 661BB9
[TV] Patched!
[HC] match1 = 1CF96
[HC] match2 = 1D047
[HC] writeAt = 1CC8E
[HC] cleanup = 1D058-1D06D
[HC] Patched!
PenMenuSystemTrayManager::GetDynamicSystemTrayHeightForMonitor() = 3C1580
Setup twinui.pcshell functions done
Setup combase functions done
Setup twinui functions done
Setup stobject functions done
Setup bthprops functions done
Setup pnidui functions done
Setup shell32 functions done
Setup windows.storage functions done
[IME] Context menu patch status: 0
Setup inputswitch functions done
Setup windowsudk.shellcommon functions done
Setup peopleband functions done
Installed hooks.
Play startup sound thread...
Signal shell ready...
Started "Signal shell ready" thread.
Open Start on monitor thread
Started "Open Start on current monitor" thread.
EP Service Window thread
Initialized taskbar centering module.
Registered Win+A, Win+B, and Win+N
Progman: 1573586
[Positioning] Added settings for monitor 0000000000010001 : 0
Progman hook: 18874987
>>> Signal shell ready.
Setup sndvolsso functions done
Ended "Signal shell ready" thread.
0 0
Started "Play startup sound" thread.
Ended "Play startup sound" thread.
[Updates] Starting daemon.
[Updates] Configured update policy on this system: "Check for updates but let me choose whether to download and install them".
[Updates] Path to module: C:\WINDOWS\dxgi.dll
[Updates] Checking against hash "22622.2428.59.1.cab5b1a83f7a0a8f"
[Updates] Update URL: https://api.github.com/repos/valinet/ExplorerPatcher/releases?per_page=1
[Updates] Prerelease update URL: "https://github.com/valinet/ExplorerPatcher/releases/download/22621.2428.59.1_ac268b1/ep_setup.exe"
[Updates] Release notes URL: "https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2428.59.1_ac268b1"
[Updates] Hash of remote file is "22622.2428.59.1.cab5b1a83f7a0a8f" (valid).
[Updates] Local version obtained from hash is 22622.2428.59.1.
[Updates] No updates are available.
[EnsureXAML] 63 ms.
10000 46000000
0 46000000

```
tell me if I need to include anything else.

**Comment by Amrsatrio** [comment]
The old system tray has been nuked in this version and I am in the process of reimplementing it. Please check the Discord server if you want to know the progress. If you want a working system tray, for now please downgrade to any other build than Canary. I will transferring this to discussions because there are a lot more other important issues to be aware of.

**Reply by CorruptionVR** [reply]
question. what discord server?

**Reply by Amrsatrio** [reply]
We have a Discord server now, the invite link is in the README.

**Reply by CorruptionVR** [reply]
oh, ok

**Comment by Amrsatrio** [comment]
My current progress is nothing more but a broken clock button:

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/7edbff55-093a-448a-baa2-4b94349bfa71)

**Comment by Mcanroe** [comment]
Sorry to add on to this discussion but i didnt want to create a new one in case this is the issue im facing as well.
Basically none of my taskbar flyout are working anymore;  battery , wifi , sound  , all dead.

Start is set to be as windows 10 style and all these flyouts are also set as the same. If i switch it to windows 7 style it works but windows 10 doesnt.

Windows build 22621.1992
EP version 22621.2361.58.4

Let me know if i need to add any other information.

**Reply by pyrates999** [reply]
Your windows version does not have the latest updates.  It is from the updates released in July.  Please check windows update for updates.

**Comment by mpoirier101** [comment]
Just got this notification this morning. Not "preview"...  Will Explorer Patcher still work ?

<img width="296" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/21262107/c1c2e9c5-aea2-4dd3-bb4a-9184a35af6d9">

Thanks !!!

**Reply by pyrates999** [reply]
In the settings app, if you have unticked "Get the latest updates as soon as available", and the update is prompting you to install it, it is a preview update.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Reply by mpoirier101** [reply]
I use Group Policy configuration to only notify of ALL/ANY updates:

<img width="560" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/21262107/54b54d6f-c53a-4c6f-996d-d5d25584c597">

It does not mention "preview" at all in the notification. I don't think this is a "preview" of 23H2... 

I will leave this update on stand-by until confirmation that Explorer Patcher works with this version.

**Reply by pyrates999** [reply]
Group policy doesn't have that option.  The settings app does.

**Reply by mpoirier101** [reply]
Group Policy is better otherwise, version 23H2 ( NOT Preview ) would have been installed...

**Reply by pyrates999** [reply]
Group policy is not available on the home version for windows.

**Comment by YisuiDenghua** [comment]
Also, Windows 10 start menu doesn't work on unsupported version 26016

**Reply by pyrates999** [reply]
Canary builds are not supported because the code is not final and can change with each new release.

**Reply by milestprower92** [reply]
All the windows 10 remanets are being removed on canary so don't expect a lot from it

**Comment by YisuiDenghua** [comment]
Seems works fine on unsupported version 23606

**Reply by milestprower92** [reply]
is only on canary builds
you're using dev

**Comment by Amrsatrio** [comment]
24H2 is now officially supported with the latest prerelease version 66.4.

**Reply by kyrcy** [reply]
I still have this problem with 22621.3880.66.4 on 24H2 (10.0.26100.1150). Is this version of Windows not supported?

**Reply by Amrsatrio** [reply]
Supposed to be supported. What is the problem you're having?

**Reply by kyrcy** [reply]
Under properties, customize notification area icons, icons are listed, but Always show all icons and notifications on the taskbar can no be ticked and "Some settings are managed by your system administrator." appears as above.

**Reply by Amrsatrio** [reply]
Please post a screenshot showing Windows 10 (ExplorerPatcher) taskbar being selected and the control panel window.

**Reply by kyrcy** [reply]
I am a little confused about how this is supposed to work. Are these the screenshots you are asking?
<img width="607" alt="taskbar" src="https://github.com/user-attachments/assets/2b8319a8-7839-4ed0-823c-a46f7a9ef622">
<img width="984" alt="control panel" src="https://github.com/user-attachments/assets/5153e1b4-5bc9-4cb7-8521-be4faac17117">

**Reply by Amrsatrio** [reply]
When the Windows 10 (ExplorerPatcher) taskbar is active and visible on screen, the checkbox works. Then you close the control panel window using the close button not with the cancel button to avoid crashing.

**Reply by pyrates999** [reply]
You can drag and drop the icon in the system tray to the popup menu that you don't want shown.

mentioned here: https://github.com/valinet/ExplorerPatcher/issues/1454

After build 25921 in windows 11, they don't have the system tray

already submitted here: https://github.com/valinet/ExplorerPatcher/discussions/2363
